src/api/routes/interviews.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * API for interviews.
+ * An interview is always bound to a user.
+ * Therefore, every endpoint requires an authentication.
+ */
+
+module.exports = (db, security) => {
+    const router = require('express').Router()
+    const validate = security.validate()
+    const messageKeys = require('../util/message-keys')
+
+    /**
+     * Get all interviews of a user.
+     * If now interviews are found, an empty array is being returned.
+     */
+    async function getInterviews(req, res) {
+        const userId = req.user.data.id
+        try {
+            const interviews = await db.getInterviews(userId)
+            if (interviews === null || !Array.isArray(interviews)) {
+                return res.status(404).json({
+                    message: 'The passed user id ("' + userId + '") was not found in the database',
+                    key: messageKeys.INVALID_USER_ID
+                })
+            }
+            return res.send(interviews)
+        }
+        catch(e) {
+            return res.status(500).json({
+                message: 'A database error occurred.',
```

Comment:
Usage of punctuation marks is inconsistent across response messages.

* `message: 'A database error occurred.'`
* `message: 'The passed user id ("' + userId + '") was not found in the database'`

Reply:
Messages without "." (dot).